src/assets/locales/resources_en_US.json

```diff
@@ -31,6 +31,30 @@
     "_description": "Label for the repeater item checkbox for the repeater component",
     "message": "Select row"
   },
+  "skyux_repeater_item_reorder_cancel": {
+    "_description": "Accessible screen reader text for when a reorder is finished",
+    "message": "Item rewording was canceled. The item was returned to its original position."
+  },
+  "skyux_repeater_item_reorder_finish": {
+    "_description": "Accessible screen reader text for when a reorder is finished",
+    "message": "Dropped the item in position"
```

Comment:
Should there be anything after `position` in this file to populate the position number? Or is that handled somewhere else? Same question below on line 48.

Reply:
Due to how we have to populate the position in the string it can't be injected into the string like some other variables can. The addition of the position number is done dynamically by the component's code.